hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/ReplicationLogQueue.java

```diff
@@ -0,0 +1,93 @@
+package org.apache.hadoop.hbase.replication.regionserver;
```

Comment:
License

Reply:
@saintstack  This is still in draft mode. Wanted to run tests first before I submit for review. Thank you !